### PR TITLE
staging → master for v1.80.0

### PR DESCRIPTION
**Pull request by CodyFinn-Akamai**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2022-11-14] - v1.80.0
+
+### Added:
+- Restore paste functionality for Glish
+- Enable Cobertura coverage report
+- add top margin
```
Comment (mjac0bs) [inline CHANGELOG.md:12]: Nitpicking inconsistencies with capitalization in the changelog.

Otherwise, testing functionality looked good.

**Review by mjac0bs — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (CodyFinn-Akamai) [reply CHANGELOG.md:12]: Oh no I forgot to paste the updated changelogs! Good catch!

**Review by mjac0bs — COMMENTED** [review]
List of changes look good now.

Just noticed the date. Do we use the date that we start the release process, or the date of the actual completed release, in the situation where there's a delay? (Today is 2022-11-15.)

**Comment by CodyFinn-Akamai** [comment]
> List of changes look good now.
> 
> Just noticed the date. Do we use the date that we start the release process, or the date of the actual completed release, in the situation where there's a delay? (Today is 2022-11-15.)

Normally we don't take two days to do a release so I think staying with the start date is fine.

**Review by dwiley-akamai — APPROVED** [review]

**Pull request merged**